mw-webapp/src/logic/wayPage/WayPage.tsx

```diff
@@ -16,7 +20,7 @@ export const WayPage = () => {
       />
       <Button
         value="Create new day report"
-        onClick={DayReportDAL.createNewDayReport}
+        onClick={() => DayReportDAL.createDayReport(uuid!)}
```

Comment:
Let's forbid to use '!' Operator by linter

Reply:
ADd rule and deleted !